wp-content/themes/cf-voyager/lessons-cpt.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Template Name: Lessons CPT
+ *
+ */
+
+get_header();
+
+$custom_terms = get_terms('lesson_types');
+
+foreach ($custom_terms as $custom_term) {
+	wp_reset_query();
+	$args = array(
+		'post_type' => 'cpt_lessons',
+		'posts_per_page' => 5,
+		'post_status' => 'publish',
+		'tax_query' => array(
+			array(
+				'taxonomy' => 'lesson_types',
+				'field' => 'slug',
+				'terms' => $custom_term->slug,
+			),
+		),
+	);
+}
+
+$the_query = new WP_Query($args);
+
+?>
+
+<?php
```

Comment:
These lines are easier to read as a one-liner: `<?php if ($the_query->have_posts()) : ?>`